x-pack/plugins/security_solution/server/lib/timeline/routes/import_timelines_route.ts

```diff
@@ -56,7 +59,7 @@ export const importTimelinesRoute = (
         body: buildRouteValidation(ImportTimelinesPayloadSchemaRt),
       },
       options: {
-        tags: ['access:securitySolution'],
+        tags: ['access:siem'],
```

Comment:
You need to explain to me how this works, I got confused with it. When I did the renaming of the app.

Reply:
Nah, it's my bad, I messed up when merging.